### ISSUE TITLE
Kakapo is not picking up its Wikidata -> wikipedia entry

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/tree-build#55]
We've had various problems with the kakapo, _Strigops habroptila_, which mainly originate in the OpenTree. I have tried overriding the OpenTree taxonomy file with the NCBI entries in 

https://github.com/OneZoom/tree-build/blob/main/data/OZTreeBuild/AllLife/BespokeTree/SupplementaryTaxonomy.tsv

But the script seems not to be able to map ncbi:57251 to the correct Wikidata entry [Q179959](https://www.wikidata.org/wiki/Q179959). We should try to figure out how/why this is going wrong, as we don't have a wikipedia entry for this at the moment.

**Comment by hyanwong** [comment]
I'm going to hack this on the prod DB

**Comment by hyanwong** [comment]
Similarly, the other override in [SupplementaryTaxonomy.tsv](https://github.com/OneZoom/tree-build/blob/main/data/OZTreeBuild/AllLife/BespokeTree/SupplementaryTaxonomy.tsv), Emberiza_calandra_ott6520265 , is not picking up its IUCN code properly. I suspect this is linked.

**Comment by hyanwong** [comment]
Ah, that could indeed be the problem.

Does the same go for Emberiza_calandra_ott6520265?

**Comment by davidebbo** [comment]
> Does the same go for Emberiza_calandra_ott6520265?

It doesn't look like it. https://www.wikidata.org/wiki/Q27220 points to ncbi 460200, which is what's in the supplementary file. So maybe something else going on here.

**Comment by davidebbo** [comment]
As for Emberiza calandra, I'm not seeing the issue, as the leaves csv has:

    899641,899641,Emberiza calandra,,6520265,27220,393215,22721020,45515755,2830.3,213701.44,14008,,460200,,,,2491505,

So it has the correct wiki ID of Q27220. Can you check your output csv? Just run:

    grep "Emberiza calandra" data/output_files/ordered_leaves_[id].csv

**Comment by hyanwong** [comment]
> Can you check your output csv?

Yes, I get that too. I've just realised that I might need to run `OZprivate/ServerScripts/Utilities/IUCNquery.py` again. I can't remember when I did that, or if the OTTs were the same when I did it.

**Comment by davidebbo** [comment]
> I've just realised that I might need to run `OZprivate/ServerScripts/Utilities/IUCNquery.py` again

I don't really understand the end to end flow. Isn't IUCNquery.py to get the latest conservation status, but I would think not necessary to get the entry to properly link to Wikipedia?

**Comment by hyanwong** [comment]
> Isn't IUCNquery.py to get the latest conservation status

That's right. I think it's only the IUCN status that's missing from the production tree. The Wikidata link seems to be OK: 

https://www.onezoom.org/life/@Emberiza_calandra=6520265?pop=ol_6520265&otthome=%40Emberiza_calandra%3D6520265#x204,y471,w1.0109

**Comment by davidebbo** [comment]
> The Wikidata link seems to be OK

I thought you had patched the DB, but maybe not for this one. It's also missing an image. Did it have one before?

In any case, it's worth running `IUCNquery` again.

So in terms of actual fixes to this repo, I think it's just changing the Kākāpō ID in `SupplementaryTaxonomy.tsv`. Maybe best if you do it directly?

**Comment by hyanwong** [comment]
Yep, I'll do that and if I can also fix the corn bunting IUCN status, I'll close this issue. Thanks for looking into it.

**Comment by hyanwong** [comment]
Closing as I have verified that re-running the IUCN script fixes Emberiza calandra. The lack of image is expected, as the harvester was using a previous OTT. This should be fixed once we gate. wikimedia harvester running (this would be a good test)
